jupyter-docker/run-jupyter-local.sh

```diff
@@ -14,6 +14,7 @@ start () {
     echo "Starting Jupyter server container..."
     docker create -it --name ${CONTAINER} --rm -p 8001:8001 $DOCKER_IMG
     docker cp jupyter-docker/jupyter_notebook_config.py ${CONTAINER}:/etc/jupyter/jupyter_notebook_config.py
+    docker cp jupyter-docker/jupyter_localize_extension.py ${CONTAINER}:/etc/jupyter/custom/jupyter_localize_extension.py
```

Comment:
I think I'm missing appending `/etc/jupyter/custom` to `PYTHONPATH` in the local version. But I also don't know why we're using `jupyter/minimal-notebook` here rather than our own image.

Reply:
This version is just for testing Jupyter locally. We don't use our image because it has dependencies on Hadoop (i.e. it expected to be installed on a Dataproc cluster). Jupyter doesn't startup if Hadoop isn't there.